ha/setup/ha_setup.py

```diff
@@ -358,9 +360,11 @@ def process(self):
                 try:
                     self._create_cluster(cluster_name, cluster_user, cluster_secret, node_name)
                     self._create_resource(s3_instances=s3_instances, mgmt_info=mgmt_info)
+                    self._alert_resource.create_alert()
                     self._confstore.set(f"{const.CLUSTER_CONFSTORE_NODES_KEY}/{node_name}")
                 except Exception as e:
                     Log.error(f"Cluster creation failed; destroying the cluster. Error: {e}")
+                    self._alert_resource.delete_alert()
```

Comment:
We need to call this in cleanup as well?

Reply:
it is not needed if cluster deleted it will be deleted also
i added here but optional

also cleanup run in parallel so we may not able to delete it in parallel like resource